DHT.cpp

```diff
@@ -33,6 +33,7 @@ float DHT::readTemperature(bool S) {
       	
       return f;
     case DHT22:
+      break;
```

Comment:
This break will prevent the code on lines 38 and 39 to be executed in case of a DHT22 sensor.
The original code executes lines 38 and 39 both in case of DHT22 *AND* in case of DHT21. With this change no data will be returned for a DHT22.